src/stdio_mgr/stdio_mgr.py

```diff
@@ -153,12 +194,12 @@ def getvalue(self):
 
 
 @contextmanager
-def stdio_mgr(in_str=""):
+def stdio_mgr(in_str="", close=True):
```

Comment:
Need `encoding` kwarg here, too.

Further: are there potentially applications where a user would want different `encoding` settings for each mocked stream? E.g., `encoding_dict={'stdin': 'utf-8', 'stdout': 'ascii'}`.

Reply:
This can be done by the caller on Python 3.7 with `.reconfigure()` , and a hack for prior versions.  We could implement `.reconfigure()` on our classes for versions prior to Python 3.7 to give the users a consistent feel.

I expect the eventual default used will be taken from the real sys objects, as that will mean stdio-mgr can be spliced into existing logic and would roughly do-the-right-thing.

Reply:
> I expect the eventual default used will be taken from the real sys objects, as that will mean stdio-mgr can be spliced into existing logic and would roughly do-the-right-thing.

Do you want to do this as part of this PR? Or, leave for later as possibly YAGNI? If the latter, probably should add this as a new issue?

Reply:
Definitely later, and https://github.com/bskinn/stdio-mgr/issues/33 is the first challenge.